app/models/audit/AuditTaskInteractionTable.scala

```diff
@@ -261,6 +261,218 @@ object AuditTaskInteractionTable {
+         |  extract(minute from SUM(diff)) +
+         |  extract(hour from SUM(diff)) * 60 AS decimal(10,2)) / 60.0 AS hours_audited
+         |FROM (
+         |  SELECT (timestamp - LAG(timestamp, 1) OVER(ORDER BY timestamp)) AS diff
+         |  FROM audit_task_interaction_small
+         |  WHERE (timestamp AT TIME ZONE 'US/Pacific') > (NOW() AT TIME ZONE 'US/Pacific') - interval '1 day'
+         |) "time_diffs"
+         |WHERE diff < '00:05:00.000' AND diff > '00:00:00.000';""".stripMargin
+    ).first
+  }
+
+  /**
+   * Calculates the total hours audited in the past week.
+   */
+  def countPastWeekAuditing(): Float = db.withSession { implicit session =>
+    Q.queryNA[Float](
+      s"""SELECT CAST(extract(second from SUM(diff)) / 60 +
+          |            extract(minute from SUM(diff)) +
+          |            extract(hour from SUM(diff)) * 60 AS decimal(10,2)) / 60.0 AS hours_audited
+          |FROM (
+          |    SELECT (ati.timestamp - LAG(ati.timestamp, 1) OVER(ORDER BY ati.timestamp)) AS diff
+          |    FROM audit_task_interaction_small ati
+          |    WHERE (ati.timestamp AT TIME ZONE 'US/Pacific') > (NOW() AT TIME ZONE 'US/Pacific') - interval '168 hours'
+          |) AS time_diffs
+          |WHERE diff < '00:05:00.000' AND diff > '00:00:00.000';
+          |""".stripMargin
+    ).first
+  }
+
+
+  /**
+   * Calculate combined time spent validating for all users using interaction logs.
+   *
+   * To do this, we take the important events from the validation_task_interaction table,
+   * get the difference between each consecutive timestamp, filter out the timestamp diffs that are greater than five
+   * minutes, and then sum those time diffs.
+   */
+  def countTotalValidating(): Float = db.withSession { implicit session =>
+    Q.queryNA[Float](
+      s"""SELECT CAST(extract(second from SUM(diff)) / 60 +
+          |            extract(minute from SUM(diff)) +
+          |            extract(hour from SUM(diff)) * 60 AS decimal(10,2)) / 60.0 AS hours_validated
+          |FROM (
+          |    SELECT (timestamp - LAG(timestamp, 1) OVER(ORDER BY timestamp)) AS diff
+          |    FROM validation_task_interaction
+          |) "time_diffs"
+          |WHERE diff < '00:05:00.000' AND diff > '00:00:00.000';""".stripMargin
+    ).first
+  }
+
+  /**
+   * Calculate combined time spent validating for all users in the past 24 hours.
+   */
+  def countTodayValidating(): Float = db.withSession { implicit session =>
+    Q.queryNA[Float](
+      s"""SELECT CAST(extract(second from SUM(diff)) / 60 +
+          |            extract(minute from SUM(diff)) +
+          |            extract(hour from SUM(diff)) * 60 AS decimal(10,2)) / 60.0 AS hours_validated
+          |FROM (
+          |    SELECT (timestamp - LAG(timestamp, 1) OVER(ORDER BY timestamp)) AS diff
+          |    FROM validation_task_interaction
+          |    WHERE (timestamp AT TIME ZONE 'US/Pacific') > (NOW() AT TIME ZONE 'US/Pacific')
```

Comment:
You didn't even subtract the 24 hours here, so this is always going to give us no data!